examples/advanced/nlp-ner/README.md

```diff
@@ -0,0 +1,74 @@
+# Federated Natural Language Processing
+
+## Introduction 
+This example shows how to use [NVIDIA FLARE](https://nvidia.github.io/NVFlare) for Natural Language Processing (NLP) tasks.
+It uses a [BERT](https://github.com/google-research/bert) model from [huggingface](https://huggingface.co/bert-base-uncased)  on a Named Entity Recognition (NER) task using the [NCBI disease](https://pubmed.ncbi.nlm.nih.gov/24393765/) dataset. 
+In this example, we select [BERT-base-uncased](https://huggingface.co/bert-base-uncased) model. 
+
+This example is adapted from this [repo](https://github.com/PL97/federated-multi-modality-learning/). 
+
+## Setup
+Install required packages for training
+```
+pip install --upgrade pip
+pip install -r ./requirements.txt
+```
+
+## Download and Preprocess Data 
+
+The data can be downloaded at the [official page](https://www.ncbi.nlm.nih.gov/CBBresearch/Dogan/DISEASE/). 
+After downloading and unzipping, it will result in three individual `.txt` files. 
+The preprocessed csv-files can be downloaded from [here](https://drive.google.com/drive/folders/13wROtEAnMgWpLMIGHB5CY1BQ1Xe2XqhG).
+
+We then use the preprocessed data to generate random splits for a 4-client experiment. 
+Please modify the `DATASET_ROOT` below to point to folder containing the four downloaded csv-files.
+```commandline
+python3 utils/split.py --data_path DATASET_ROOT --num_clients 4
```

Comment:
Can probably remove the `utils` folder and have the split.py in the root dir of the example. The test files should be moved to unit testing.